_data/nav.yml

```diff
@@ -0,0 +1,10 @@
+head:
+    - Home
+    - title: Install
+      url: https://pysal.readthedocs.io/en/latest/installation.html
```

Comment:
Not Sure if there is the best way to present the installation of pysal? The page which is linked to only introduces how to install the metapackage but not including the installation of subpackages.

Reply:
We could have an install.md page that outlines the choices of going with the meta-package install or the installation of specific packages, with links to the instructions for the latter.